core/services/workflows/engine.go

```diff
@@ -321,6 +325,7 @@ func (e *Engine) init(ctx context.Context) {
 
 	if retryErr != nil {
 		e.logger.Errorf("initialization failed: %s", retryErr)
+		logCustMsg(e.cma, fmt.Sprintf("initialization failed: %s", retryErr), e.logger)
```

Comment:
Can we call this "registration failed" rather than "initialization failed" so we're consistent with the naming